<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditA</class>
 <widget class="QMainWindow" name="EditA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>edita</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFontComboBox" name="SelectFont">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>229</width>
      <height>23</height>
     </rect>
    </property>
    <property name="currentFont">
     <font>
      <family>微软雅黑</family>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="TextSelectFont">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>字体：</string>
    </property>
   </widget>
   <widget class="QLabel" name="TextFontSize">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>53</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>字体大小：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="FontSize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>340</y>
      <width>42</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <number>13</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="Edit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>653</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助 </string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件 </string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_save"/>
    <addaction name="action_open"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>编辑 </string>
    </property>
    <addaction name="action_copy"/>
    <addaction name="action_pause"/>
    <addaction name="action_shear"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="status"/>
  <widget class="QToolBar" name="tool">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="action_about">
   <property name="text">
    <string>关于 &quot;EditA&quot;</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>新建文档</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出 EditA</string>
   </property>
  </action>
  <action name="action_copy">
   <property name="text">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_pause">
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_shear">
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
